### PR TITLE
[New] Style point with distance composite scene symbol

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR implements `Style point with distance composite scene symbol` in `Scenes` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-StylePointWithDistanceCompositeSceneSymbol/Shared/Samples/Style%20point%20with%20distance%20composite%20scene%20symbol)

## Linked Issue(s)

- `swift/issues/4899`

## How To Test

- Zoom in and out to see the symbol change.

## Screenshots

style-point-with-distance-composite-scene-symbol

### CODE REVIEW

File: Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            let graphicsOverlay = GraphicsOverlay()
+            graphicsOverlay.sceneProperties.surfacePlacement = .relative
+            return graphicsOverlay
+        }()
+        
+        /// The plane graphic created from a distance composite symbol.
+        private let planeGraphic: Graphic = {
+            // Create the different symbols.
+            let planeSymbol = ModelSceneSymbol(url: .bristol, scale: 100)
+            let coneSymbol = SimpleMarkerSceneSymbol.cone(color: .red, diameter: 200, height: 600)
```
Comment (dfeinzimer) [inline Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift:89]: Should this use a centered anchor? The cone seems to be positioned a bit higher than the plane and the jump is noticeable.

```suggestion
            let coneSymbol = SimpleMarkerSceneSymbol.cone(color: .red, diameter: 200, height: 600, anchorPosition: .center)
```

File: Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            let distanceCompositeSymbol = DistanceCompositeSceneSymbol()
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: planeSymbol, minDistance: 0, maxDistance: 10000)
+            )
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: coneSymbol, minDistance: 10001, maxDistance: 30000)
+            )
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: circleSymbol, minDistance: 30001, maxDistance: 0)
+            )
```
Comment (dfeinzimer) [inline Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift:103]: Just a suggestion here, take or leave as desired

```suggestion
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: planeSymbol, maxDistance: 10000)
            )
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: coneSymbol, minDistance: 10001, maxDistance: 30000)
            )
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: circleSymbol, minDistance: 30001)
            )
```

File: Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                cameraController.cameraPitchOffset = 80
+                cameraController.cameraHeadingOffset = -30
+                return cameraController
+            }()
+        }
+    }
+}
+
+private extension URL {
+    /// A URL to the local Bristol 3D model files.
```
Comment (dfeinzimer) [inline Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift:130]:
```suggestion
    /// A URL to the local Bristol 3D model file.
```

File: Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    static var bristol: URL {
+        Bundle.main.url(forResource: "Bristol", withExtension: "dae", subdirectory: "Bristol")!
+    }
+    
+    /// A world elevation service from Terrain3D ArcGIS REST service.
```
Comment (dfeinzimer) [inline Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift:135]:
```suggestion
    /// A world elevation service from the Terrain3D ArcGIS REST service.
```

**Review by dfeinzimer — COMMENTED** [review]
Just a few things.

**Review by rolson — APPROVED** [review]

**Review by dfeinzimer — DISMISSED** [review]

**Review by rolson — APPROVED** [review]

**Review by dfeinzimer — APPROVED** [review]

**Comment by CalebRas** [comment]
Thanks you guys!

**Pull request merged**